tests/e2e/Shared/gateways.js

```diff
@@ -0,0 +1,49 @@
+const banktransfer = {
```

Comment:
This for now shows only part of the gateways, this will need to be probably taking the details from the config